tests/test_spark_connect.py

```diff
@@ -0,0 +1,20 @@
+import chispa
```

Comment:
Because you modified a `spark.py` we should be able to run all the existing tests against PySpark-Connect... In this case it seems it me, that we do not need that file.

Reply:
Thanks, @SemyonSinchenko. Does this mean we should be applying this to all the test cases in `Quinn`? 

There are 3 test cases that fail, I can work through to resolve them in this PR and come back. Let me know?

Reply:
Let's focus only in incorporating CI in this PR? And all the failed tests should be fixed in next PRs. What do you think?

Reply:
Makes sense, @SemyonSinchenko . Avoids creating a monolithic PR. I will remove the `test_spark_connect.py` and the calls to that from the ci.yml and the Makefile as well. 

Will work on another PR that will call the test cases from the CI for both `spark-classic` and `spark-connect`. Are you okay with that?

Reply:
That sounds great for me! Thank you!